### PR TITLE
fix(circleci): unbreak plugin

### PR DESCRIPTION
**Pull request by Nek**
## Hey, I just made a Pull Request!

The CircleCI Plugin build info page got broken. Fixed it.

#### :heavy_check_mark: Checklist
- [x] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: plugins/circleci/src/pages/BuildsPage/BuildsPage.tsx [hunk 1]
```diff
@@ -19,13 +19,18 @@ import { Grid } from '@material-ui/core';
...
+import { AppStateProvider } from '../../state/AppState';
+import { Settings } from '../../components/Settings';
 
 const BuildsPage: FC<{}> = () => (
-  <Layout>
-    <Content>
-      <Builds />
-    </Content>
-  </Layout>
+  <AppStateProvider>
```
Comment (shmidt-i) [inline plugins/circleci/src/pages/BuildsPage/BuildsPage.tsx:26]: Doesn't this mean that state (e.g. settings,cache) is going to be invalidated between different pages?

**Review by shmidt-i — COMMENTED** [review]
Could you please elaborate on what the problem was? Not quite sure what these changes are aimed towards

**Review by freben — APPROVED** [review]

File: plugins/circleci/src/pages/BuildsPage/BuildsPage.tsx (hunk 1 above)
Comment (Nek) [reply plugins/circleci/src/pages/BuildsPage/BuildsPage.tsx:26]: Fixing.

**Comment by Nek** [comment]
> Could you please elaborate on what the problem was? Not quite sure what these changes are aimed towards

Not really. I had to choose between a quick fix and actual deeper investigation of the problem. Superficially the routing wasn't hitting the build page and was showing an empty page. I couldn't figure out why.

**Pull request merged**